HealthKitReporter/Classes/Model/Payload/Quantity.swift

```diff
@@ -25,6 +25,7 @@ public struct Quantity: Identifiable, Sample {
         }
     }
 
+    public let uuid: UUID
```

Comment:
please make it String. This is a potential use in Flutter, which does not accept UUID type - will be required additional changes in Code Bridging files.

```suggestion
    public let uuid: String
```